package.json

```diff
@@ -28,15 +28,15 @@
     "build:web": "vite build",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
-    "icons": "cd resources && npx png2icons ../static/favicon.png icon -allwe -bc"
+    "icons": "cd resources && npx png2icons ../static/favicon.png icon -allwe -bc",
+    "postinstall": "npx base-volta-off-of-nwjs@latest"
```

Comment:
After every `npm install` this will run and check what version of NW.js is in your `node_modules` then check what version of Node.js it uses. It will then set the `volta.node` value in `package.json`.